### ISSUE TITLE
`ppdzmin` should be optional

### ISSUE THREAD

**Issue by malmans2** [pyNEMO/pyDOMCFG#44]
`ppdzmin` is not used for uniform grids, so I think it should become an optional argument.
@oceandie what do you think?

**Comment by malmans2** [comment]
Actually, I think `pphmax` as well is not necessarily used? For example when the grid is not uniform and `ppsur, ppa0, ppa1` are not computed?

**Comment by oceandie** [comment]
For `ppdzmin` I agree. For `pphmax` no, it is used also for uniform grids, see [here](https://github.com/pyNEMO/pyDOMCFG/blob/main/pydomcfg/domzgr/zco.py#L223)

**Comment by malmans2** [comment]
Maybe I'm just getting confused, but I think that if grid is NOT uniform and `ppsur, ppa0, ppa1` are specified, `pphmax` is not used.
I.e., in that scenario the user would have to assign a dummy value to `pphmax`

**Comment by malmans2** [comment]
For example, in our ORCA2 test you can assign any value to `ppdzmin` and `pphmax` and the test will pass
https://github.com/pyNEMO/pyDOMCFG/blob/f68612d139e5b405d08e04172ca3257b3d77d1b6/pydomcfg/tests/test_zco.py#L32-L34

**Comment by oceandie** [comment]
> 
> 
> Maybe I'm just getting confused, but I think that if grid is NOT uniform and `ppsur, ppa0, ppa1` are specified, `pphmax` is not used.
> I.e., in that scenario the user would have to assign a dummy value to `pphmax`

Yes, in the case `ppsur, ppa0, ppa1` are specified (computed) pphmax is not used. Sorry, I didn't understand what you meant before with 'ppsur, ppa0, ppa1 are not computed' .
